connector/client/client.go

```diff
@@ -33,6 +37,8 @@ func NewClient(config *Config) (Client, error) {
 }
 
 // Post a activity to given URL
+// Creates a HTTP POST request with the provided activity as the body and a Bearer token in the header
+// Returns any error as received from the call to connector service
```

Comment:
Add full stop or empty line in between. On godoc it renders as a single line. This happened at multiple places.